test/Microsoft.AspNetCore.SignalR.Redis.Tests/TestConnectionMultiplexer.cs

```diff
@@ -0,0 +1,295 @@
+        }
+
+        public string GetStormLog()
+        {
+            throw new NotImplementedException();
+        }
+
+        public ISubscriber GetSubscriber(object asyncState = null)
+        {
+            return _subscriber;
+        }
+
+        public int HashSlot(RedisKey key)
+        {
+            throw new NotImplementedException();
+        }
+
+        public long PublishReconfigure(CommandFlags flags = CommandFlags.None)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<long> PublishReconfigureAsync(CommandFlags flags = CommandFlags.None)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void RegisterProfiler(IProfiler profiler)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void ResetStormLog()
+        {
+            throw new NotImplementedException();
+        }
+
+        public void Wait(Task task)
+        {
+            throw new NotImplementedException();
+        }
+
+        public T Wait<T>(Task<T> task)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void WaitAll(params Task[] tasks)
+        {
+            throw new NotImplementedException();
+        }
+    }
+
+    public class TestSubscriber : ISubscriber
+    {
+        private static ConcurrentDictionary<RedisChannel, List<Action<RedisChannel, RedisValue>>> _globalSubscriptions =
+            new ConcurrentDictionary<RedisChannel, List<Action<RedisChannel, RedisValue>>>();
+
+        private ConcurrentDictionary<RedisChannel, Action<RedisChannel, RedisValue>> _subscriptions =
+            new ConcurrentDictionary<RedisChannel, Action<RedisChannel, RedisValue>>();
+
+        public ConnectionMultiplexer Multiplexer => throw new NotImplementedException();
+
+        public EndPoint IdentifyEndpoint(RedisChannel channel, CommandFlags flags = CommandFlags.None)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<EndPoint> IdentifyEndpointAsync(RedisChannel channel, CommandFlags flags = CommandFlags.None)
+        {
+            throw new NotImplementedException();
+        }
+
+        public bool IsConnected(RedisChannel channel = default)
+        {
+            throw new NotImplementedException();
+        }
+
+        public TimeSpan Ping(CommandFlags flags = CommandFlags.None)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<TimeSpan> PingAsync(CommandFlags flags = CommandFlags.None)
+        {
+            throw new NotImplementedException();
+        }
+
+        public long Publish(RedisChannel channel, RedisValue message, CommandFlags flags = CommandFlags.None)
+        {
+            if (_globalSubscriptions.TryGetValue(channel, out var handlers))
+            {
+                foreach (var handler in handlers)
+                {
+                    handler(channel, message);
+                }
```

Comment:
Should you return `handlers.Count` here?

Reply:
I guess